### PR TITLE
Feature/new navbar notification

### PR DESCRIPTION
**Pull request by JDownloader**

### CODE REVIEW

**Comment by antleblanc** [comment]
Related to: ovh-ux/ovh-manager-dedicated#368.

File: client/components/navbar/navbar-notification.service.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+class NavbarNotificationService {
```
Comment (FredericEspiau) [inline client/components/navbar/navbar-notification.service.js]: It looks like this code is duplicated in all 4 managers, right ? Maybe it should be a component in its own repository

Comment (whispyy) [reply client/components/navbar/navbar-notification.service.js]: Yeah, in fact it's already a component, this is the service that handle the configuration of it.

Comment (JDownloader) [reply client/components/navbar/navbar-notification.service.js]: So is all the code for the Navbar and Sidebar, it has already been determined it was a faster and less elegant approche.

Comment (JDownloader) [reply client/components/navbar/navbar-notification.service.js]: It cannot be the same, translator and constants are not the same across all managers.

**Pull request merged**